app/view/show.py

```diff
@@ -11,8 +11,33 @@
 from app.pacman import get_pkg
 from app.util import chunks, multiline_to_list
 from collections import defaultdict
+from urllib.parse import urlencode
 
 
+def link_encode(url, query):
+    query = {k: v for k, v in query.items()}
+    data = urlencode(query)
+    return "%s?%s" % (url, data)
+
+
+def add_bug_link(cves, pkgs):
+    bug_url = "https://bugs.archlinux.org/newtask"
+
+    summary_desc = '<give a short summary about all CVEs>'
```

Comment:
this should definitively be moved to a template, that way its lot easier to maintain the format and add text related changes.

something like:
``` templates/bug.txt ```